### ISSUE TITLE
does it work ?

### ISSUE THREAD

**Issue by PManager1** [hereisethanhunt/GmailOAuth2#1]

**Comment by hereisethanhunt** [comment]
Yes, it works. Place your client secret and client id in the .js file. you will be able to authenticate using your gmail account

**Comment by PManager1** [comment]
Can the logged in user send email once he’s logged in ?
